pycobertura/reporters.py

```diff
@@ -174,6 +174,28 @@ def generate(self):
         )
 
 
+class PhorgeJsonReporter(Reporter):
+    """
+    A reporter that outputs a JSON object in the Phorge/Phabricator
+    coverage format. It is required to have the source files available.
+    """
+
+    def generate_one(self, filename):
+        statuses = self.cobertura.line_statuses(filename)
+        with self.cobertura.filesystem.open(filename) as f:
+            lines = ["N" for _ in f]
+            for lineNum, covered in statuses:
+                lines[lineNum - 1] = "C" if covered else "U"
```

Comment:
I have a long overdue PR that will introduce partially covered lines. Is there a character you'd like to use when I merge this PR eventually?

Reply:
As per phorge docs, it does not support partially covered lines.